virttest/utils_hotplug.py

```diff
@@ -0,0 +1,401 @@
+#
+# library for hotplug(cpu) related helper functions
+# can be extended to memroy related helper functions aswell
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; specifically version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: IBM (c) 2017
+# Author: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>
+
+
+import re
+import json
+import logging
+import platform
+
+from . import virsh
+from . import libvirt_xml
+from . import utils_misc
+from . import utils_test
+from utils_test import libvirt
+from libvirt_xml.xcepts import LibvirtXMLNotFoundError
+from autotest.client import utils
```

Comment:
Sure, have sent a avocado utils PR https://github.com/avocado-framework/avocado/pull/2010 once it is merged will change this, let have it till then,